client/components/Scenario/index.jsx

```diff
@@ -34,7 +34,10 @@ class Scenario extends Component {
     }
 
     isScenarioRun() {
-        return location.pathname.includes('/moment/');
+        return (
```

Comment:
this should be a prop - perhaps even pass the run id to `<Scenario run={id} />`

Reply:
Done